radial_hit.m

```diff
@@ -76,14 +77,25 @@
 
 if (isempty(time))
     tMaxR = inf;
+    tStepR = -1;
```

Comment:
Why -1? Shouldn't this just be 0? I know in theory this step will never occur since ```tMaxR = inf```, but should still be 0 if I'm not mistaken.